AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/OperationTests/HostedUITests/AWSAuthHostedUISignInTests.swift

```diff
@@ -69,137 +69,112 @@ class AWSAuthHostedUISignInTests: XCTestCase {
             hubEventHandler: MockAuthHubEventBehavior())
     }
 
-    func testSuccessfulSignIn() {
+    @MainActor
```

Comment:
Why do we need main actor here?

Reply:
Creating the presentation anchor needs to be on the main UI thread otherwise the unit test logs a warning and will fail.